prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="Home Page.html"> Click here to go to the home page
+  </a>
```

Comment:
Hi Sixx, when you click on this link in the browser, it should load the home page. Try an `href` attribute of `"../index.html"`